src/darwin/Framework/CHIP/MTRDeviceController_Internal.h

```diff
@@ -219,6 +219,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (MTRDevice *)_setupDeviceForNodeID:(NSNumber *)nodeID prefetchedClusterData:(nullable NSDictionary<MTRClusterPath *, MTRDeviceClusterData *> *)prefetchedClusterData;
 - (void)removeDevice:(MTRDevice *)device;
 
+/**
+ * Clear all delegates from the device controller
+ */
+- (void)clearDeviceControllerDelegates;
```

Comment:
We don't need this in the header, right?  It's only used inside the implementation, and could be called `_clearDeviceControllerDelegates`?

Reply:
Of course!